projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/yes-no/read-yes-no-field.component.ts

```diff
@@ -4,7 +4,7 @@ import { YesNoService } from './yes-no.service';
 
 @Component({
   selector: 'ccd-read-yes-no-field',
-  template: `<span class="text-16">{{formattedValue ? (caseField.label | rpxTranslate : null : formattedValue) : null}}</span>`
+  template: `<span class="text-16">{{formattedValue ? (caseField.value | rpxTranslate : null : formattedValue) : null}}</span>`
```

Comment:
It feels like you ought to be using formattedValue here, not value. They are almost always the same, but since the null check is against formattedValue, you should logically use that